src/app/features/tech-record/components/tech-record-summary/tech-record-summary.component.ts

```diff
@@ -76,6 +77,7 @@ export class TechRecordSummaryComponent implements OnInit {
   vehicleTechRecordCalculated!: TechRecordModel;
   sectionTemplates: Array<FormNode> = [];
   dtpNumbersFromRefData: FormNodeOption<string>[] = [];
+  middleIndex!: number;
```

Comment:
middleIndex is not always defined

Reply:
middleIndex would not be used unless there was section templates, I which case it would be set. I did however initialise it to 0 to avoid doubt.